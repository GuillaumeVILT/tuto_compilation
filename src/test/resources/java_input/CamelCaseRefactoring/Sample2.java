
public class Sample2 {

	public void not_camel_case_method_1() {

	}

	public void notcamel_case_method2() {

	}

	public void alreadyCamelCaseMethod() {

	}

	public int _4times8() {
		return 4*8;
	}
}