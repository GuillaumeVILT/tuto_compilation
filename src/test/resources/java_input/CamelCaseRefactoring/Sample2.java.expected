public class Sample1 {
  public void notCamelCaseMethod1(){
  }
  public void notcamelCaseMethod2(){
  }
  public void alreadyCamelCaseMethod(){
  }
  public int _4times8(){
    return 4 * 8;
  }
}
